eng/tools/typespec-validation/test/sdk-tspconfig-validation.test.ts

```diff
@@ -184,6 +184,27 @@ const tsManagementGenerateMetadataTestCases = createEmitterOptionTestCases(
   [new TspConfigTsMgmtModularGenerateMetadataTrueSubRule()],
 );
 
+const newTsManagementGenerateMetadataTestCases = createEmitterOptionTestCases(
+  "@azure-tools/typespec-ts",
+  managementTspconfigFolder,
+  "generateMetadata",
+  true,
+  false,
+  [new TspConfigTsMgmtModularGenerateMetadataTrueSubRule()],
+);
+
+const mixTsManagementGenerateMetadataTestCase = {
```

Comment:
is this case for when mixing happened, `generate-metadata` one wins?

Reply:
Yes